### PR TITLE
DrawerKit 0.4.1 release

### PR DESCRIPTION
**Pull request by wltrup**
- Reverses the fix for the [issue](https://github.com/Babylonpartners/DrawerKit/issues/31) about safe areas. The fix broke other things (even caused crashes in certain scenarios) and the issue will need to be re-opened.

- Changes the minimum deployment target to 10.2.

### CODE REVIEW

File: DrawerKit/DrawerKit.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -343,7 +343,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.3;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.2;
```
Comment (zzcgumn) [inline DrawerKit/DrawerKit.xcodeproj/project.pbxproj]: Is this change intentional?

Comment (wltrup) [reply DrawerKit/DrawerKit.xcodeproj/project.pbxproj]: Yes. It makes it easier to test `DrawerKit` in the simulator.

File: DrawerKit/DrawerKit/Internal API/AnimationController.swift [hunk 2]
```diff
@@ -28,10 +28,7 @@ extension AnimationController: UIViewControllerAnimatedTransitioning {
 
     func animateTransition(using transitionContext: UIViewControllerContextTransitioning) {
         let (presentingVC, presentedVC) = viewControllers(transitionContext, isPresentation)
-        if isPresentation {
-            let controller = presentedVC.presentationController as! PresentationController
-            controller.intermediate.view.addSubview(presentedVC.view)
-        }
+        if isPresentation { transitionContext.containerView.addSubview(presentedVC.view) }
```
Comment (zzcgumn) [inline DrawerKit/DrawerKit/Internal API/AnimationController.swift]: Our normal coding style is
```
if condition {
    do something
}
```
also for simple one-liners.

Comment (wltrup) [reply DrawerKit/DrawerKit/Internal API/AnimationController.swift]: Fixed in https://github.com/Babylonpartners/DrawerKit/pull/52/commits/b5e3bd79da24ca127c88c8f442ddde4282d6c73d.
